docker-compose.yml

```diff
@@ -1,14 +1,15 @@
-version: "2.1"
+---
+version: "3.8"
 services:
  app:
-   build: .
-   networks:
-     - main
+   image: wikipedialibrary/externallinks:latest
```

Comment:
the tag could just be an variable defined in `.env` and substituted here.

Reply:
Done!